### ISSUE TITLE
Documentation - The code font size should be larger

### ISSUE THREAD

**Issue by AAstridVH** [Avaiga/taipy-doc#4]
**Description**
For accessibility purpose, the font size of the code extracts should be determined in (1)em and have a minimum fallback size of 16px.

**Acceptance Criteria**
- [ ] Ensure new code is unit tested, and check code coverage is at least 90%
- [ ] Propagate any change on the demos and run all of them to ensure there is no breaking change
- [ ] Ensure any change is well documented

**Comment by AAstridVH** [comment]
After discussing the matter, we should push the site to 18px (1.1250em).
